services/agora/src/components/ui-library/ZKTab.vue

```diff
@@ -1,22 +1,38 @@
 <template>
-  <div class="textStyle" :class="{ highlightTab: isHighlighted }">
-    {{ text }}
+  <div class="tabStyle" :class="{ highlightTab: isHighlighted }">
+    <!--  TODO: proper icon color -->
+    <!-- :color="isHighlighted ? 'primary' : '#7D7A85'" -->
+    <ZKIcon
+      v-if="iconCode !== undefined"
+      :color="isHighlighted ? '#6b4eff' : '#7D7A85'"
```

Comment:
@e32wong 
I'd love you to help me figure that out properly. Was not able to use `primary`, had to resort to hardcode the color value...